HalClient.Net.Tests/HalClient.Net.Tests.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project=".\..\HalClient.Net.Build\HalClient.Net.Commons.props" />
   
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFrameworks>netcoreapp1.1;net462</TargetFrameworks>

```

Comment:
This is just so we can run tests that run on the .NET Standard 2.0-targeted DLL.

Reply:
Why do we need net462 explicitly here? To be honest i'm not that familiar with the `<TargetFramework>` setting (https://docs.microsoft.com/en-us/dotnet/standard/frameworks), but imho we should just set it to `netstandard1.3` instead. That would make it compatible with both `netcoreapp` and full framework right?

Reply:
Adding .NET 4.6.2 as a build target in the test project makes it possible to test the .NET Standard 2.0 build of the HAL client library too.

In the test runner (Resharper's in this case), it looks like this:

![image](https://user-images.githubusercontent.com/32968/56950607-2b472680-6b36-11e9-8447-74017809c205.png)